### DISCUSSION TITLE
shadow quality.

### DISCUSSION THREAD

**Discussion by ghost** [RavEngine/RavEngine#9] [General]
how do i increase the shadow quality i was testing the samples and noticed that the shadows were really low quality...
![image](https://github.com/RavEngine/RavEngine/assets/118131071/68f9d88a-6125-4d1a-8319-42cef4e96b0c)

**Comment by Ravbug** [comment]
There's no API for it yet. You can modify https://github.com/RavEngine/RavEngine/blob/ba4313e8168d54a1dc1b158f8b6ea93baef5000b/src/RenderEngine.cpp#L79 in the meantime to increase the resolution.

But note that the render engine is currently a very early WIP and so is missing many features compared to other game engines.

**Reply by ghost** [reply]
so once i change that value it will increase the resolution the next frame?
if so i will make a fork and make the change and then make a pull request

**Reply by Ravbug** [reply]
No it's a hardcoded constant at the moment, changing the value at runtime won't do anything. A mechanism for changing the resolution at runtime is in-the-works but I can't give any ETA.

**Reply by ghost** [reply]
i will try my best to implement it.

**Reply by ghost** [reply]
yea... ok i dont know what i'm doing when i change the resolution to anything but 2048 i get no shadows....
